src/sentry/integrations/vercel/integration.py

```diff
@@ -76,6 +77,85 @@ def get_organization_config(self):
 
         return fields
 
+    def update_organization_config(self, data):
+        # data = {"project_mappings": [[sentry_project_id, vercel_project_id]]}
+
+        metadata = self.model.metadata
+        vercel_client = VercelClient(metadata["access_token"], metadata.get("team_id"))
+        config = self.org_integration.config
+        sentry_project_id = data["project_mappings"][-1][0]
```

Comment:
@ceorourke We are assuming that the last project is the one being added but this is not going to work if a project is removed as this function is called there as well. You don't necessarily have to update it now if you don't want but that does mean this function will have to be updated to work with that case in the future. If that's the course you want to take, probably need a comment.